connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -43,6 +44,14 @@ message Command {
   }
 }
 
+message SqlCommand {
```

Comment:
Can you explain the semi-eager evaluation mode here?
- An actual command is executed & its result is returned as a long relation.
- Any other query, we don't return anything.

Reply:
Done.